packages/plugins/i18n/server/src/register.ts

```diff
@@ -14,21 +14,44 @@ export default ({ strapi }: { strapi: Strapi }) => {
   addContentTypeSyncHooks(strapi);
 };
 
+/**
+ * Match urls for model creation
+ *  /content-manager/collection-types/api::category.category/
+ *  /content-manager/collection-types/api::category.category
+ * And not match:
+ *  /content-manager/collection-types/api::category.category/1
+ *  /content-manager/collection-types/api::category.category/1/actions/publish
+ */
+const isUrlForCreation = (url: string) => {
+  if (!url) return false;
+
+  // Split path and remove empty strings
+  // [ 'content-manager', 'collection-types', 'api::category.category' ]
+  const splitUrl = url.split('/').filter(Boolean);
+
+  // Get the last element of the array
+  // api::category.category / 1 / publish
+  const model = splitUrl[splitUrl.length - 1];
+
+  // If the model contains :: it means it's a uid
+  return model.includes('::');
```

Comment:
@innerdvations just in case, all uids do have the XX::XX.XX format right?

Reply:
This is one thing I'm worried about. I'm trying to think if there's a better way to do this, it seems a bit fragile

Reply:
what about this:

```ts
  const splitUrl = url.split('/').filter(Boolean);
  return split.Url.lenght === 3
  ```

Reply:
is it possible for people to have prefixes in their CM url?

Reply:
```suggestion
return mode instanceof UID_FROM_TYPES
```
Is something like that possible?

Reply:
I would prefer not to, just in case people use models that are not stored in strapi